src/test/java/bio/terra/service/filedata/DrsTest.java

```diff
@@ -271,44 +268,6 @@ public void drsHackyTest() throws Exception {
             datasetIamRoles.get(IamRole.SNAPSHOT_CREATOR)));
   }
 
-  @Test
-  public void drsScaleTest() throws Exception {
```

Comment:
https://github.com/DataBiosphere/jade-data-repo/pull/1566 covers this functionality in unit tests, so I removed this integration test:
- We confirm that `DrsMetricsService.tryIncrementCurrentDrsRequestCount` caps the number of concurrent requests
- We confirm that the above method is called during DRS resolution